oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceConfigurationTest.java

```diff
@@ -83,7 +84,9 @@ public void defaultValues() throws Exception {
         assertEquals("MONGO", config.documentStoreType());
         assertEquals(DocumentNodeStoreService.DEFAULT_BUNDLING_DISABLED, config.bundlingDisabled());
         assertEquals(DocumentMK.Builder.DEFAULT_UPDATE_LIMIT, config.updateLimit());
-        assertEquals(Arrays.asList("/"), Arrays.asList(config.persistentCacheIncludes()));
+        assertEquals(of("/"), Arrays.asList(config.persistentCacheIncludes()));
```

Comment:
nitpick : looks unrelated/unnecessary?

Reply:
Yes, it can be avoided, but IntelliJ insisted on making this change.